### ISSUE TITLE
False positive for UTF-8 in tag contents

### ISSUE THREAD

**Issue by oliverklee** [sds/haml-lint#53]
In one of our HAML templates (which still mostly is a click dummy), we have the following construct:

```
%p
  Freue dich auf einen erfolgreichen Workshop! Alkoholische Getränke sind
  zu Übungszwecken bereitgestellt und im Preis enthalten.
```

https://gist.github.com/oliverklee/cd3ae86204661cb83847

This causes the following warning in haml-lint:
app/views/home/index.html.haml:2 [W] RuboCop: Style/AsciiComments: Use only ascii symbols in comments.

It looks like haml-lint passes the contents of the p tag as a Ruby comment to RuboCop.
